core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -82,6 +83,16 @@ public LoggingDecoratorBuilder logger(Logger logger) {
         return this;
     }
 
+    /**
+     * Sets ths {@link Logger} to use when logging using loggerName.
```

Comment:
Sets the specified `loggerName` which is used to create a new {@link Logger}.

Reply:
Thanks 😄